renku/core/commands/view_model/composite_plan.py

```diff
@@ -71,13 +71,15 @@ def __init__(
         self,
         id: str,
         name: str,
+        derived_from: str,
         mappings: List[ParameterMappingViewModel],
         links: List[ParameterLinkViewModel],
         steps: List[StepViewModel],
         description: Optional[str] = None,
     ):
         self.id = id
         self.name = name
+        self.derived_from = derived_from
```

Comment:
Is this used in `renku.cli`? I don't see a place where it is displayed. Since we currently treat immutability as a behind-the-scenes thing, it probably shouldn't be on the viewmodel.